library/src/main/java/org/xmtp/android/library/Conversation.kt

```diff
@@ -99,13 +124,16 @@ sealed class Conversation {
                             .setKeyMaterial(conversationV2.keyMaterial.toByteString()),
                     ),
             ).build()
+
+            is Group -> throw XMTPException("Groups do not support topics")
```

Comment:
Why is `TopicData` and `keyMaterial` exposed in v2? What will integrators do with it?

Reply:
I believe `TopicData` and `keyMaterial` is used for push notifications. That way they can export from react native or javascript and import into iOS or android and not need to reload the entire costly `conversation.list()` method to decrypt a push notification.